shared/app.js

```diff
@@ -73,7 +74,9 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
Its allows us to statically require a different override without having to have a custom include path. I also want to start moving everything to optional includes like this so rendr can be pluggable, being able to include things like react-router and such in the future.